ECHO-COVID19.ipynb

```diff
@@ -228,158 +206,11 @@
    },
    "outputs": [],
    "source": [
```

Comment:
Line 218/196 needs to be modified for clarity: "the orange line shows the average number of tests 2000-2020"

Reply:
This needs to be checked for all such chart intros

Reply:
Just to be clear, it's 2001-2020 inclusive. I made edits to make that clearer.